Modules/Core/ImageFunction/include/itkLinearInterpolateImageFunction.hxx

```diff
@@ -51,8 +51,6 @@ LinearInterpolateImageFunction<TInputImage, TCoordinate>::EvaluateUnoptimized(co
 
   // When RealType is VariableLengthVector, 'value' will be resized properly
   // below when it's assigned again.
-  Concept::Detail::UniqueType<typename NumericTraits<RealType>::ScalarRealType>();
```

Comment:
I am not sure what is this supposed to assert, so I don't know whether it is fine to remove it. This seems to be the only instance of its use?